io/aio-wrapper.cpp

```diff
@@ -366,17 +366,18 @@ namespace photon
     };
     static thread_local AioResetHandle *reset_handler = nullptr;
 
-    int libaio_wrapper_init()
+    int libaio_wrapper_init(int iodepth)
     {
         if (libaio_ctx)
             return 0;
 
         std::unique_ptr<libaio_ctx_t> ctx(new libaio_ctx_t);
         ctx->evfd = eventfd(0, EFD_NONBLOCK | EFD_CLOEXEC);
+        libaio_ctx->iodepth = iodepth > IODEPTH_MAX ? IODEPTH_MAX : iodepth;
```

Comment:
libaio_ctx->iodepth should be >= 1 ？

Reply:
once iodepth <1, initial will be failed.
just let the initial failure tells user that parameter is nat a valid value.

Reply:
int io_setup(unsigned nr_events, aio_context_t *ctx_idp);